rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -195,7 +195,8 @@ public void getRecoveringLRAs(Map<URI, Transaction> lras) {
 
     /**
      * remove an LRA log record
-     * @param lraUid
+     *
+     * @param lraUid lra id that will be removed from the log record
```

Comment:
LRA is an acronym so it should be capitalised (ie lra -> LRA) but obviously this comment does not apply to java variable names.

Reply:
+1